src/validators/output/index.js

```diff
@@ -1,7 +1,10 @@
-import {noop} from 'lodash'
+import {isPlainObject, noop} from 'lodash'
+import {isAbsolute} from 'path'
+
 export default [
-  {key: 'output.filename', validate: noop},
-  {key: 'output.path', validate: noop},
+  {key: 'output', validate: validateOutput},
```

Comment:
One problem with adding a validator on the `output` object is we lose the ability to tell whether someone's provided a property that's invalid. See [this](https://github.com/kentcdodds/configuration-validator/blob/kentcdodds-patch-1/README.md#uncovered-path-validation) for more info on why that is and why it matters :-)